control/cli.py

```diff
@@ -132,7 +132,7 @@ def connect(self, nvme_config):
         # Bind the client and the server
         self._stub = pb2_grpc.NVMEGatewayStub(channel)
         # Set up logging
-        self._logger = nvme_config.logger
+        self._logger = settings.logger
```

Comment:
It seems the logger does not have any interaction with any settings, so maybe move it out of the settings class.